### PR TITLE
#1043 - Add Flow Container Margins And Break Options

### PR DESCRIPTION
**Pull request by Guilherme-Almeida-Zeni**
## Changes
Added hability to set margin / break in flow container components.

## Related Issues
Fixes #1043

### CODE REVIEW

File: packages/bodiless-layouts/src/FlowContainer/useGetMenuOptions.tsx [hunk 1]
```diff
@@ -206,6 +206,64 @@ const useSwapButton = (
...
+  props: EditFlowContainerProps,
+  item: FlowContainerItem,
+) => {
+  const context = useEditContext();
+  const { components } = withNoDesign(props);
+  const isActive = item.margin;
+  const { updateFlowContainerItem } = handlers;
+  const handler = () => {
+    // Toogles margin prop in flow container item.
+    const newItem: FlowContainerItem = isActive ? omit(item, 'margin') : { ...item, margin: 'mb-10' };
```
Comment (VancheeZze) [inline packages/bodiless-layouts/src/FlowContainer/useGetMenuOptions.tsx:220]: Is there any way we can make this margin configurable?

File: packages/bodiless-layouts/src/FlowContainer/utils/flowContainerUtils.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+ * getBreakTag provides HTML markup for non-visible HR element in case
+ * the given flow container component is enabled to break rows.
+ *
+ * @param item Flow container item
+ *
+ * @returns Non-visible HR Tag or, if break is disabled, empty string
+ */
+export const getBreakTag = (item: FlowContainerItem) => {
+  const className = 'w-full border-transparent border-0';
+  return (item.break) ? <Hr className={className} /> : '';
```
Comment (VancheeZze) [inline packages/bodiless-layouts/src/FlowContainer/utils/flowContainerUtils.tsx:15]: Should we return `null` instead of an empty string here?

**Review by VancheeZze — DISMISSED** [review]

File: packages/bodiless-layouts/src/ComponentMargin/componentMarginForm.tsx [hunk 3]
```diff
@@ -23,28 +23,39 @@ import uiContext, { defaultUI } from './uiContext';
...
-  const finalUI:FinalUI = { ...defaultUI, ...useMenuOptionUI(), ...ui };
+
+  const finalUI: FinalUI = { ...defaultUI, ...useMenuOptionUI(), ...ui };
 
   return (
     <uiContext.Provider value={finalUI}>
       <finalUI.MasterWrapper className="flex flex-col">
-        <finalUI.MarginSideCheckboxesWrapper>
-          <finalUI.MarginSideCheckboxLabel htmlFor="Bottom">
+        <finalUI.MarginSideCheckboxesWrapper className="bl-my-grid-1">
```
Comment (VancheeZze) [inline packages/bodiless-layouts/src/ComponentMargin/componentMarginForm.tsx:36]: Is there a way we can add classes to the `ui` components themselves before passing these components to the `ui`? Similar to `ComponentSelector` or other components in the `bodiless-layouts-ui` package?

**Comment by dewen** [comment]
Confirmed to close

**Pull request closed without merging**